src/main/resources/jenkins/plugins/git/GitSSHPrivateKeyBinding/help-credentialsId.html

```diff
@@ -0,0 +1,31 @@
+<div>
+    Set the git private key for <strong>SSH</strong> protocol.
+
+<p>
+<strong>Shell example</strong>
+<pre>
+withCredentials([GitUsernamePassword(credentialsId: 'my-credentials-id', gitToolName: 'git-tool')]) {
```

Comment:
Start with lower case letter
```suggestion
withCredentials([gitUsernamePassword(credentialsId: 'my-credentials-id', gitToolName: 'git-tool')]) {
```